### ISSUE TITLE
Better icons for train restrictions

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#17]
See: https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/issues/127

Should be relatively trivial to implement (just changing some graphics) and would give a big usability improvement.

I can do graphic editing so if you can point me to where the files are I can see if I can do it and submit a PR (or, with my git repo skills, probably just upload new graphics to this issue ticket lol)

**Comment by krzychu124** [comment]
I saw that yesterday while I was browsing issues ;) In your repository folder: 'Cities-Skylines-Traffic-Manager-President-Edition/TLM/TLM/Resources/'. There you can find all graphics of mod. 
You can put updated icons to zip archive and post it even here, no problem :)

**Comment by aubergine10** [comment]
In addition to the ```*_allowed.png``` & ```*_restricted``` icons, there are also ```*_infosign.png``` icons.

For example: ```passengertrain_infosign.png```.

Are ```_infosign``` icons used by TMPE or can I safely ignore them? I suspect they are just original icons that Victor used when creating the allowed/restricted icons...?

FYI, I intend to create extra folders inside TLM/TLM/Resources:

* templates = template images for allow/deny backgrounds
* originals = original high res icons should anyone need them at later date
* thumbnails = smaller versions of various icons for use in documentation

LMK if there are any issues with creating those subfolders.

**Comment by aubergine10** [comment]
Also, I notice the allowed/restricted icons are 800x800 px -- this seems somewhat excessive, considering their on-screen size?  I suspect they could be resized to 400x400 without any negative impacts on image quality in-game.

**Comment by krzychu124** [comment]
I have no issues with those folders, I can easily exclude them while building new version.

Regarding to `*_infosign.png` icons, unfortunately you can't ignore them because they are used while you're creating a complex Timed Traffic lights e.g. on road segment with monorail junction and some others IIRC, and these signs have a bit different shape as you can see 😉

**Comment by aubergine10** [comment]
Ah, in that case I'll need to update the train _infosign icons. Will do that tomorrow and send over a PR.
